### ISSUE TITLE
Dominican Republic is not the the database

### ISSUE THREAD

**Issue by DannyFeliz** [SaraVieira/uber-cities#42]
It seems like [Dominican Republic](https://isthereuber.in/dominican-republic) is not in the databases. Uber has been in the Dominican Republic for more than 2 years, you can see the site for the Dominican Republic here -> [Uber Dominicana Republic](https://www.uber.com/do/en/)

**Comment by SaraVieira** [comment]
In https://www.uber.com/en-DO/cities/santo-domingo ?

**Comment by DannyFeliz** [comment]
@sarahdayan Yes, also [Santiago](https://www.uber.com/en-DO/cities/santiago-rd) and [Puerto Plata](https://www.uber.com/en-DO/cities/puerto-plata)
